### PR TITLE
Only create vHosts if certificate exists

### PR DESCRIPTION
**Pull request by andreaswolf**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -28,6 +28,8 @@ apache_vhosts_ssl: []
   #   # Optional.
   #   certificate_chain_file: "/path/to/certificate_chain.crt"
 
+apache_ignore_missing_ssl_certificate: True
```
Comment (geerlingguy) [inline defaults/main.yml]: Just a style note; though I may change this at some point, for now I'm using lowercase bool definitions in all my roles, so can you update this to `true`?

File: tasks/configure-Debian.yml [hunk 2]
```diff
@@ -23,6 +23,11 @@
   with_items: apache_mods_disabled
   notify: restart apache
 
+- name: Stat SSL certificates.
```
Comment (geerlingguy) [inline tasks/configure-Debian.yml]: Just to make this a little more clear, can you name it `name: Check whether certificates defined in vhosts exist.`?

File: README.md [hunk 3]
```diff
@@ -69,6 +69,10 @@ No SSL vhosts are configured by default, but you can add them using the same pat
         certificate_chain_file: "/path/to/certificate_chain.crt"
       }
 
+If you would like to only create vHosts when the certificate is available (e.g. when using Let’s encrypt), set
```
Comment (geerlingguy) [inline README.md]: As the new var is meant to be a top-level variable, please move this documentation down below the existing SSL vhosts documentation, like:

```
    apache_ignore_missing_ssl_certificate: true

If you would like to only create SSL vhosts when the vhost certificate is present (e.g. when using Let’s Encrypt), set `apache_ignore_missing_ssl_certificate` to `false`. When doing this, you might need to run your playbook more than once so all the vhosts are configured (if another part of the playbook generates the SSL certificates).
```

**Comment by andreaswolf** [comment]
Updated according to your comments, @geerlingguy.

**Pull request merged**